docs/about.md

```diff
@@ -0,0 +1,50 @@
+| [`sdk-conformance`](https://github.com/oscal-compass/sdk-conformance)                 |                                                                                                                          |
+
+### Demo content repositories - OSCAL in _action_
+| Demo Content                                                                                  | Description                                                                                             |
+|-----------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------|
+| [`e2e-demo`](https://github.com/oscal-compass/e2e-demo)                                       |                                                                                                         |
+| [`e2e-demo-ssp`](https://github.com/oscal-compass/e2e-demo-ssp)                               |                                                                                                         |
+| [`e2e-demo-cd`](https://github.com/oscal-compass/e2e-demo-cd)                                 |                                                                                                         |
+| [`e2e-demo-profile`](https://github.com/oscal-compass/e2e-demo-profile)                       | Repository to demo OSCAL based profile and agile authoring using compliance-trestle and GitHub actions. |
+| [`e2e-demo-catalog`](https://github.com/oscal-compass/e2e-demo-catalog)                       | Repository to demo OSCAL based catalog and agile authoring using complaince-trestle and GitHub actions. |
+| [`e2e-demo-mapping`](https://github.com/oscal-compass/e2e-demo-mapping)                       |                                                                                                         |
+| [`e2e-demo-xccdf-ubuntu`](https://github.com/oscal-compass/e2e-demo-xccdf-ubuntu)             |                                                                                                         |
+| [`e2e-demo-compliance-posture`](https://github.com/oscal-compass/e2e-demo-compliance-posture) |                                                                                                         |
+| [`compliance-trestle-demos`](https://github.com/oscal-compass/compliance-trestle-demos)       | Demo setup for compliance-trestle.                                                                      |
+| [`compliance-trestle-ssp-demo`](https://github.com/oscal-compass/compliance-trestle-ssp-demo) | Demonstration of compliance trestle's SSP authoring capabilities.                                       |
+| [`compliance-trestle-fedramp`](https://github.com/oscal-compass/compliance-trestle-fedramp)   | Compliance trestle plugin to support FedRAMP specific functionality.                                    |
```

Comment:
`compliance-trestle-fedramp` is a plugin from `trestle`. Maybe a new section for this?